src/main/java/io/dropwizard/oor/healtcheck/OorHealthCheck.java

```diff
@@ -35,11 +37,14 @@ public OorHealthCheck(boolean status) {
 
     @Override
     protected Result check() throws Exception {
+        Result result = null;
         if(oor.get()) {
-            return Result.unhealthy("Service is out of rotation");
+            result = Result.unhealthy("Service is out of rotation");
         } else {
-            return Result.healthy();
+            result = Result.healthy();
         }
+        log.debug(String.format("OORBundle HealthCheck status: %s, %s", result.isHealthy(), result.getMessage()));
```

Comment:
Maybe use inbuilt string formatter for composing the log line (`{}`)

Reply:
Done.